### PR TITLE
Version 1.0.2 with clean-css 5.1.5

### PR DESCRIPTION
**Pull request by jakubpawlowicz**

### CODE REVIEW

**Comment by jakubpawlowicz** [comment]
Hope I did it right @johannchopin

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2] - 2021-08-05
+### Changed
+- Updates clean-css dependency to version 5.1.5
+
 ## [1.0.1] - 2021-07-29
 ### Changed
 - Updates clean-css dependency to version 5.1.4
```
Comment (johannchopin) [inline CHANGELOG.md:15]: Concerning the CHANGELOG please keep the previous versions description and add the links to the reference (my bad I forgot them in first place):

```suggestion
## [1.0.2] - 2021-08-05
### Changed
- Updates clean-css dependency to version 5.1.5

## [1.0.1] - 2021-07-29
### Changed
- Updates clean-css dependency to version 5.1.4


[Unreleased]: https://github.com/clean-css/clean-css.github.io/compare/v1.0.2...HEAD
[1.0.2]: https://github.com/clean-css/clean-css.github.io/releases/tag/v1.0.2
[1.0.1]: https://github.com/clean-css/clean-css.github.io/releases/tag/v1.0.1
```

Comment (jakubpawlowicz) [reply CHANGELOG.md:15]: No problem. Should I commit your suggestion? Or perhaps we merge PR and then you make Changelog change on main branch?

Comment (johannchopin) [reply CHANGELOG.md:15]: No you can directly merge the change in this PR I would say 👍

**Review by johannchopin — APPROVED** [review]
CR ✅ Thanks a lot 💪

**Pull request merged**